README.md

```diff
@@ -46,7 +46,7 @@ A repo for documentation and code involved in setting up different dbt Learn exp
   }
   ```
 
-  - The dbt_cloud_token is accessible from dbt Cloud under Your Profile > Personal profile > API
+  - The dbt_cloud_token is accessible by Creating a new Personal access token (PAT) from dbt Cloud under Your Profile > API tokens > Personal tokens. Pay attention to create a PAT and not a Service token. Your PAT should start with a dbtu_ prefix.
```

Comment:
suggestion: add a piece about creating this PAT in the sandbox account for the GT since PATs will change account to account.